### DISCUSSION TITLE
Icon "code" instead of "picture" in Settings

### DISCUSSION THREAD

**Discussion by KYCowboy** [netalertx/NetAlertX#789] [Q&A] [answered]
I'm just getting started with Docker and NetAlertX, so this may be a  dumb question, but when I look at a particular setting such as NEWDEV_dev_Icon, what I see appears to be a code snippet.  I have no idea what the icon actually looks like.

![Clipboard_09-10-2024_01](https://github.com/user-attachments/assets/7c83e063-5c11-45f0-a5c1-32a4c8e323b7)

Is there any way to convert the snippet to an image?  Could this be done within Settings?

Many thanks!

- Tim -

**Comment by jokob-sk** [comment]
Hi @KYCowboy ,

This will be availabel in the next release. You can also test this in the `netalertx-dev` image. Backup everything first, just to be sure.

j

**Reply by KYCowboy** [reply]
Looks great!  This maybe pushing my luck, but could you add the same functionality to Pre-defined icons (UI_ICONS)?  Just another "nice to have", so what you've already done is all I really needed.

Thanks!

- Tim -

**Reply by jokob-sk** [reply, marked as answer]
Hey @KYCowboy - I thought about it, but it's a completely different UI component so it's not like I can piggyback off the work I've done on the above. Just the above thing took me almost a day - so I'd rather not see that part of the code for a while 😅 Happy to accept PRs though if you code and want to help to make the app better 🙏

**Reply by KYCowboy** [reply]
Thank you for your consideration.  While I once was, for my sins, a programmer for IBM, that was many years ago.  Now in my dotage I just like to dink around with the marvelous and intersting products you young folks produce.  Keep up the great work!

- Tim -

**Reply by jokob-sk** [reply]
Thanks for your support  @KYCowboy ! Feel free to share the app on socials so I get more young folks to contribute  😅
